packages/l/libwfkafka/xmake.lua

```diff
@@ -1,11 +1,11 @@
 package("libwfkafka")
     set_homepage("https://github.com/sogou/workflow")
-    set_description("C++ Parallel Computing and Asynchronous Networking Engine")
+    set_description("The Kafka Client of C++ Workflow")
     set_license("Apache-2.0")
 
     add_urls("https://github.com/sogou/workflow/archive/refs/tags/$(version).tar.gz",
              "https://github.com/sogou/workflow.git")
-    add_versions("v0.10.6", "20d2177dda391676235687b03980eb50e9ea11dd")
+    add_versions("v0.10.6", "989faa6e8057b960eff713ca96e1cac810421589")
```

Comment:
This is not a git hash but a checksum (because the url has a file), and it's not right:
```
error: ...modules/private/action/require/impl/actions/download.lua:187: unmatched checksum, current hash(5[70](https://github.com/xmake-io/xmake-repo/actions/runs/4281238759/jobs/7454016384#step:5:71)1ef31) != original hash(989faa6e)
stack traceback:
    [C]: in function 'error'
    [@programdir/core/base/os.lua:912]:
    [...modules/private/action/require/impl/actions/download.lua:187]: in function '_download'
    [...modules/private/action/require/impl/actions/download.lua:322]:

  => download https://github.com/sogou/workflow/archive/refs/tags/v0.10.6.tar.gz .. failed
```

Reply:
We thought it's the commit ID…… Sorry.

Reply:
BTW, how do I  get the checksum? Is it a SHA256？

Reply:
Yes, it's the sha256 of the file content (the v0.10.6.tar.gz file you can download from github).

Reply:
Well, I'v fixed the checksum of two packages. May you check it?